python-regression/tests/features/steps/transaction_steps.py

```diff
@@ -22,26 +24,36 @@ def generate_transaction_and_attach(step, node):
     arg_list = step.hashes
```

Comment:
It was mostly for readability on my part since I don't particularly like `step.hashes`, the name doesn't really define it's purpose in any way, but i've removed it here since it is unnecessary